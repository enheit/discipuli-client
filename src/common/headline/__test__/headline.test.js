import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Headline from '../headline';

describe('Headline component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Headline title="Headline" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should be rendered with class \'.headline--large\'', function() {
    expect(shallow(<Headline title="Headline" large />)
      .is('.headline--large'))
      .toBe(true);
  });

  it('should be rendered with class \'.headline--small\'', function() {
    expect(shallow(<Headline title="Headline" small />)
      .is('.headline--small'))
      .toBe(true);
  });
});
