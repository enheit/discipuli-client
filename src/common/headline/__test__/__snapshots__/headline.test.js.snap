// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline component renders correctly 1`] = `
<h1
  className="headline"
>
  Headline
</h1>
`;
